devdocs/macos-developer-id.md

```diff
@@ -0,0 +1,12 @@
+## Expired Apple Developer ID
+
+1. To replace an expired Apple certificate, you must download the julia-buildkite repo and decrypt by running `make decrypt`.
```

Comment:
```suggestion
1. To replace an expired Apple certificate, clone the https://github.com/JuliaCI/julia-buildkite repository and decrypt it by running `make decrypt`.
```
Also, might explain how to get the `decrypt` program:
```console
% make decrypt 
cd .buildkite/cryptic_repo_root && decrypt --repo-root=$(pwd) --verbose
/bin/sh: line 1: decrypt: command not found
make: *** [Makefile:4: decrypt] Error 127
```

Furthermore, from what I understand below one needs to do this on macOS, it'd be better to make it clear from the beginning.